ubuntu/debian/control

```diff
@@ -53,6 +52,8 @@ Depends: libignition-cmake2-dev (>= 2.3.0),
          libignition-math6-dev (>= 6.6.0),
          libignition-math6-eigen3-dev (>= 6.6.0),
          libignition-plugin-dev,
+         libx11-dev,
+         freeglut3-dev,
```

Comment:
I have!
```bash
cd /home/jrivero/code/ignition/debug_ogre_next_jammy/ws/build/ignition-rendering6/src && /usr/bin/c++ -Dignition_rendering6_EXPORTS -I/home/jrivero/code/ignition/debug_ogre_next_jammy/ws/src/ign-rendering/include -I/home/jrivero/code/ignition/debug_ogre_next_jammy/ws/build/ignition-rendering6/include -I/home/jrivero/code/ignition/debug_ogre_next_jammy/ws/build/ignition-rendering6/core/include -isystem /usr/include/ignition/common4 -isystem /usr/include/uuid -isystem /usr/include/ignition/math6 -isystem /usr/include/ignition/utils1 -isystem /usr/include/ignition/plugin1 -isystem /home/jrivero/code/ignition/debug_ogre_next_jammy/ws/install/include/ignition/cmake2 -O2 -g -DNDEBUG  -Wall -Wextra -Wno-long-long -Wno-unused-value -Wfloat-equal -Wshadow -Winit-self -Wswitch-default -Wmissing-include-dirs -pedantic -UNDEBUG -fPIC -fvisibility=default -msse -msse2 -mfpmath=sse -msse3 -mssse3 -msse4.1 -msse4.2 -fPIC -I/usr/include/uuid -MD -MT src/CMakeFiles/ignition-rendering6.dir/Utils.cc.o -MF CMakeFiles/ignition-rendering6.dir/Utils.cc.o.d -o CMakeFiles/ignition-rendering6.dir/Utils.cc.o -c /home/jrivero/code/ignition/debug_ogre_next_jammy/ws/src/ign-rendering/src/Utils.cc
cd /home/jrivero/code/ignition/debug_ogre_next_jammy/ws/build/ignition-rendering6/src && /usr/bin/c++ -Dignition_rendering6_EXPORTS -I/home/jrivero/code/ignition/debug_ogre_next_jammy/ws/src/ign-rendering/include -I/home/jrivero/code/ignition/debug_ogre_next_jammy/ws/build/ignition-rendering6/include -I/home/jrivero/code/ignition/debug_ogre_next_jammy/ws/build/ignition-rendering6/core/include -isystem /usr/include/ignition/common4 -isystem /usr/include/uuid -isystem /usr/include/ignition/math6 -isystem /usr/include/ignition/utils1 -isystem /usr/include/ignition/plugin1 -isystem /home/jrivero/code/ignition/debug_ogre_next_jammy/ws/install/include/ignition/cmake2 -O2 -g -DNDEBUG  -Wall -Wextra -Wno-long-long -Wno-unused-value -Wfloat-equal -Wshadow -Winit-self -Wswitch-default -Wmissing-include-dirs -pedantic -UNDEBUG -fPIC -fvisibility=default -msse -msse2 -mfpmath=sse -msse3 -mssse3 -msse4.1 -msse4.2 -fPIC -I/usr/include/uuid -MD -MT src/CMakeFiles/ignition-rendering6.dir/WireBox.cc.o -MF CMakeFiles/ignition-rendering6.dir/WireBox.cc.o.d -o CMakeFiles/ignition-rendering6.dir/WireBox.cc.o -c /home/jrivero/code/ignition/debug_ogre_next_jammy/ws/src/ign-rendering/src/WireBox.cc
/home/jrivero/code/ignition/debug_ogre_next_jammy/ws/src/ign-rendering/src/Utils.cc:19:10: fatal error: X11/Xlib.h: No such file or directory
   19 | #include <X11/Xlib.h>
      |          ^~~~~~~~~~~~
compilation terminated.
```

Reply:
```bash
/home/jrivero/code/ignition/debug_ogre_next_jammy/ws/src/ign-rendering/src/ShaderParams_TEST.cc: In member function 'virtual void ShaderParams_ConstRangeForLoopDoesNotDirty_Test::TestBody()':
/home/jrivero/code/ignition/debug_ogre_next_jammy/ws/src/ign-rendering/src/ShaderParams_TEST.cc:69:19: warning: loop variable 'name_param' creates a copy from type 'const std::pair<const std::__cxx11::basic_string<char>, ignition::rendering::v6::ShaderParam>' [-Wrange-loop-construct]
   69 |   for (const auto name_param : params)
      |                   ^~~~~~~~~~
/home/jrivero/code/ignition/debug_ogre_next_jammy/ws/src/ign-rendering/src/ShaderParams_TEST.cc:69:19: note: use reference type to prevent copying
   69 |   for (const auto name_param : params)
      |                   ^~~~~~~~~~
      |                   &
/home/jrivero/code/ignition/debug_ogre_next_jammy/ws/src/ign-rendering/ogre/src/OgreMaterial.cc: In member function 'void ignition::rendering::v6::OgreMaterial::UpdateShaderParams(ignition::rendering::v6::ConstShaderParamsPtr, Ogre::GpuProgramParametersSharedPtr)':
/home/jrivero/code/ignition/debug_ogre_next_jammy/ws/src/ign-rendering/ogre/src/OgreMaterial.cc:386:19: warning: loop variable 'name_param' creates a copy from type 'const std::pair<const std::__cxx11::basic_string<char>, ignition::rendering::v6::ShaderParam>' [-Wrange-loop-construct]
  386 |   for (const auto name_param : *_params)
      |                   ^~~~~~~~~~
/home/jrivero/code/ignition/debug_ogre_next_jammy/ws/src/ign-rendering/ogre/src/OgreMaterial.cc:386:19: note: use reference type to prevent copying
  386 |   for (const auto name_param : *_params)
      |                   ^~~~~~~~~~
      |                   &
/home/jrivero/code/ignition/debug_ogre_next_jammy/ws/src/ign-rendering/ogre/src/OgreRenderEngine.cc:22:11: fatal error: GL/glx.h: No such file or directory
   22 | # include <GL/glx.h>
      |           ^~~~~~~~~~
compilation terminated.
```

Reply:
so I see X11 as a private dependency of the core library:

* https://github.com/ignitionrobotics/ign-rendering/blob/ignition-rendering6_6.3.0/src/CMakeLists.txt#L42-L44
* https://github.com/ignitionrobotics/ign-rendering/blob/ignition-rendering6_6.3.0/src/Utils.cc#L18-L21

but I only see `gl.h` or `glut.h` in `ogre*` or `example` files